airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/source.py

```diff
@@ -136,7 +137,7 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
         """
         try:
             stream_kwargs = self._get_stream_kwargs(config)
-            stream_to_check = VendorSalesReports(**stream_kwargs)
+            stream_to_check = Orders(**stream_kwargs)
```

Comment:
Why are you changing this?

Reply:
hello @marcosmarxm, Im sorry, I forget to explain this. It's not working, only returning 400 client error. We have all permissions in our app, Orders works like a charm.

Reply:
oh seems like, its already fixed in 1.4.1